### ISSUE TITLE
how the y-axis of the lineplot of the HINT motif footprint was calculated and normalized?

### ISSUE THREAD

**Issue by crazyhottommy** [CostaLab/reg-gen#116]
Hi,

I am reading https://www.regulatory-genomics.org/hint/tutorial/ at the bottom, two line plots were presented. How the y-axis was calculated?

Is it the average read counts for each base pair (100 bp flanking the motif) and then the average count of each base pair across all the motif occurrence was calculated? How this value is normalized?

Thanks!
Ming

**Comment by lzj1769** [comment]
Hi @crazyhottommy ,

> I am reading https://www.regulatory-genomics.org/hint/tutorial/ at the bottom, two line plots were presented. How the y-axis was calculated?

It's not simply the average count of each base pair, but the bias-corrected and normalized signal.

> How this value is normalized?

For normalization, we borrowed the idea from DEseq2, specifically, we first calculate the activity score for each combination of TF and condition and then compute the normalization factor based on median-of-ratios. Finally, the signals are divided by the noralization factor.

Best,
Li

**Comment by crazyhottommy** [comment]
Hi, is it possible to output the bias-corrected score so I can plot myself the footprint ?

**Comment by lzj1769** [comment]
> Hi, is it possible to output the bias-corrected score so I can plot myself the footprint ?

you mean the bias-corrected signal?

**Comment by crazyhottommy** [comment]
yes!

**Comment by lzj1769** [comment]
> yes!

Yes, we also output a text file for each motif which contains the signal. You can find it under Lineplots folder created by rgt-hint.

**Comment by crazyhottommy** [comment]
thanks!
